helm_deploy/values-preprod.yaml

```diff
@@ -0,0 +1,16 @@
+---
+# Per environment values which override defaults in send-legal-mail-to-prisons-api/values.yaml
+
+generic-service:
+  replicaCount: 2
```

Comment:
Number of replicas in preprod would be higher than dev generally?

Reply:
Yeah should be the same as prod... I'll give that more thought. Thanks!